test/jacvec_operators.jl

```diff
@@ -55,9 +55,9 @@ tspan = (0.0,100.0)
 ff1 = ODEFunction(lorenz,jac_prototype=JacVecOperator{Float64}(lorenz,u0))
 ff2 = ODEFunction(lorenz,jac_prototype=JacVecOperator{Float64}(lorenz,u0,autodiff=false))
 
-#These tests are behaving strangely
-
+#These tests are behaving strangely - hence commented out

```

Comment:
please don't ever comment out tests. It leads to tests getting lost. Set the block to broken if you have to, but this test passes on master so if there's a problem it's caused here.

Reply:
I've merged @YingboMa's fix for the test from master - they pass now